src/deviceApiCalls/transports/extension.transport.js

```diff
@@ -69,6 +69,8 @@ async function extensionSpecificRuntimeConfiguration (deviceApi) {
             contentScope: contentScope,
             // @ts-ignore
             userPreferences: {
+                // Copy locale to user preferences as 'language' to match expected payload
+                language: contentScope.locale,
```

Comment:
@sjbarag @shakyShane Is there a reason to deviate from the precedent of the extension (passing in `contentScope`)? If there's an existing convention used elsewhere, might as well adopt that. But I don't know how widely adopted it is, or whether there are other existing conventions for the native apps, so just asking you folks.

Reply:
yep - content-scope is usually the entire remote-config JSON blob, user preferences is specifically there to provide on-device overrides or config.